.github/workflows/LIGHTHOUSE.yml

```
@@ -27,4 +27,4 @@ jobs:
           uploadArtifacts: true
           temporaryPublicStorage: true
         env:
-          LHCI_GITHUB_APP_TOKEN: ${{ secrets.LHCI_GITHUB_APP_TOKEN }}
+          LHCI_GITHUB_APP_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

試してはいませんが、`LHCI_GITHUB_TOKEN` 環境変数を使うと動作するかもしれません。
https://github.com/GoogleChrome/lighthouse-ci/blob/875c3ed181ae410c23e7c7c784d0448587fa00b3/packages/cli/src/upload/upload.js#L158-L173

```suggestion
          LHCI_GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```